### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by duxv** [nicolasparada/nakama#45] [Ideas]
It would be great if you could create a Discord server for the development of Nakama.

**Comment by nicolasparada** [comment]
Right now there is just me doing the development.
Sure I could if there were more people.

**Reply by duxv** [reply]
I'm just a beginner, but I would really like to contribute.

**Comment by nicolasparada** [comment]
Or just use the very nakama.social for it 😏

**Reply by duxv** [reply]
Discord is more advanced. Nakama wouldn't compare to Discord, yet :)

**Comment by nicolasparada** [comment]
https://discord.gg/wkvwRa3pju 😉
